Dockerfile

```diff
@@ -28,4 +28,4 @@ WORKDIR /opt
 ENV PYTHONPATH .
 ARG DL_IMAGE_VERSION=latest
 ENV DL_IMAGE_VERSION=$DL_IMAGE_VERSION
-CMD ["/usr/bin/python", "src/app.py"]
```

Comment:
I thought a bit more about this and it's probably better to keep the app.py name for compatibility with current (and older) zalando-locust versions and create a 'locust-wrapper' symlink in /usr/local/bin What do you think?

Reply:
I am not sure about the benefit for creating symlink. "I thought a bit more about this” -> Could you tell me more about your idea?

Reply:
Since zalando-locust runs app.py by default, the current change may not be compatible. Also, there's no need to specify a path to run a symlink from /usr/locl/bin